src/test/java/com/amadeus/jenkins/plugins/workflow/libs/HttpRetrieverIntegrationTests.java

```diff
@@ -225,18 +230,23 @@ FilePath buildJobWithLibrary(@Nullable String fixtureName, @Nullable String libr
                             .willReturn(WireMock.notFound()));
         }
         globalLibraries.getLibraries().add(new LibraryConfiguration(
-                libraryName,
-                new HttpRetriever(urlBuilder.apply(libraryName), credentials.getId(), withPreemptiveAuth)
+                "foo",
+                new HttpRetriever(urlBuilder.apply("foo"), credentials.getId(), withPreemptiveAuth)
         ));
         WorkflowJob p = j.jenkins.createProject(WorkflowJob.class, "p");
         p.setDefinition(new CpsFlowDefinition(importScript, true));
 
         WorkflowRun run = j.buildAndAssertSuccess(p);
-        return new FilePath(run.getRootDir().toPath().resolve("libs").resolve(libraryName).toFile());
+        List<Path> children = Files.list(run.getRootDir().toPath().resolve("libs"))
+                .filter(Files::isDirectory)
+                .collect(Collectors.toList());
```

Comment:
this creates a stream holding a file handle - and it is never closed, this is likely a cause of the issue on windows where the clean-up after the test can not happen correctly.

Terminal operators do not close underlying streams.